packages/library-sync/src/nativeDarwin/kotlin/io/realm/kotlin/mongodb/internal/HttpClientCache.kt

```diff
@@ -19,5 +20,8 @@ internal actual class HttpClientCache actual constructor(timeoutMs: Long, custom
 }
 
 public actual fun createPlatformClient(block: HttpClientConfig<*>.() -> Unit): HttpClient {
-    return HttpClient(Darwin, block)
+    return HttpClient(Darwin) {
+        install(WebSockets)
```

Comment:
Why are we installing it here if Darwin isn't supported yet?

Also it seems to already being installed here: https://github.com/realm/realm-kotlin/pull/1528/files#diff-2270f40a8e4fabaacf4f2791bbe75dee7428db4c09f84fc606ab90aaa99a78e1R277